fdbserver/DataDistribution.actor.cpp

```diff
@@ -1591,6 +1604,34 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 		return totalHealthyMachineCount;
 	}
 
+	std::pair<uint32_t, uint32_t> calculateMinMaxServerTeamNumOnServer() {
```

Comment:
These two methods seem to be good candidate for code reuse. Non blocking totally.

Reply:
I feel there is not much too reuse. 
This function checks the server team number while the later one checked the machine team number.
Differentiating them seems introduce more complex than benefit. :)

Reply:
What I meant is something like generalizing them into one method using template + lambda. But keeping them may make things clearer. I don't have a strong opinion.